utils/tesseract/Makefile

```
@@ -0,0 +1,55 @@
+# Copyright (C) 2019 Valentin Kivachuk <vk18496@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tesseract
+PKG_VERSION:=4.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/tesseract-ocr/tesseract/tar.gz/$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=a1f5422ca49a32e5f35c54dee5112b11b99928fc9f4ee6695cdc6768d69f61dd
+
+PKG_MAINTAINER:=Valentin Kivachuk <vk18496@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+
+define Package/tesseract
+  SUBMENU:=Tesseract
```

```suggestion
  MENU:=1
```
Remove this so it does not add a menu just for the package.  With `MENU:=1`, and removing the `SUBMENU` lines from this and tessdata, the menu will still be created, inline with the tesseract package:
```
<M> tesseract............................... Tesseract Open Source OCR Engine  --->
```
Even though I'm writing here for simplicity, you may want to move the `MENU:=1` line to the end of the block to keep a consistent style.